projects/sensory_interface/public/app.js

```diff
@@ -25,4 +30,43 @@ function fillRow(values, currentTr) {
         td.appendChild(document.createTextNode(myValue));
         currentTr.appendChild(td);
     }
+}
+
+function addOnClickSoundToTable() {
+    let element;
+    for (element of document.getElementsByTagName("td")) {
+        element.addEventListener("click", startSoundPlayback);
+    }
+}
+
+function startSoundPlayback() {
+    if (context.state == "suspended") {
+        context.resume();
+    }
+    let request = new XMLHttpRequest();
+    request.open("get", "beep_digital.mp3", true);
```

Comment:
It's good practice to put files such as mp3, images etc under an `assets` subfolder.

Reply:
To bee sure, you want me to call the sub folder "assets"? or "sounds"?

Reply:
`assets`